### PR TITLE
[W5.11][T13-2]XU HANHUI

### PR DESCRIPTION
**Pull request by HansKoh**
I have changed find command from case sensitive to non-case sensitive.

### CODE REVIEW

File: src/seedu/addressbook/common/Utils.java [hunk 1]
```diff
@@ -34,4 +35,16 @@ public static boolean elementsAreUnique(Collection<?> items) {
...
+    {
+        String[] stringsArray = strings.toArray(new String[0]);
+        for (int i=0; i<stringsArray.length; ++i)
+        {
+            stringsArray[i] = stringsArray[i].toLowerCase();
+        }
+        strings.clear();
+        strings.addAll(Arrays.asList(stringsArray));
+    }
+
```
Comment (xllx1) [inline src/seedu/addressbook/common/Utils.java]: Utils is a good home for the convertToLowerCase method!

File: src/seedu/addressbook/common/Utils.java [hunk 2]
```diff
@@ -34,4 +35,16 @@ public static boolean elementsAreUnique(Collection<?> items) {
...
+    public static void convertToLowerCase(Collection<String> strings)//This can use for any other Collection data type: eg.Set
+    {
+        String[] stringsArray = strings.toArray(new String[0]);
+        for (int i=0; i<stringsArray.length; ++i)
+        {
+            stringsArray[i] = stringsArray[i].toLowerCase();
+        }
+        strings.clear();
+        strings.addAll(Arrays.asList(stringsArray));
+    }
```
Comment (xllx1) [inline src/seedu/addressbook/common/Utils.java]: Utils is a good home for the convertToLowerCase method!

File: src/seedu/addressbook/common/Utils.java [hunk 3]
```diff
@@ -34,4 +35,16 @@ public static boolean elementsAreUnique(Collection<?> items) {
         }
         return true;
     }
+
+    public static void convertToLowerCase(Collection<String> strings)//This can use for any other Collection data type: eg.Set
```
Comment (xllx1) [inline src/seedu/addressbook/common/Utils.java]: Utils is a good home for the convertToLowerCase method!

File: src/seedu/addressbook/common/Utils.java [hunk 4]
```diff
@@ -34,4 +35,16 @@ public static boolean elementsAreUnique(Collection<?> items) {
...
+    {
+        String[] stringsArray = strings.toArray(new String[0]);
+        for (int i=0; i<stringsArray.length; ++i)
```
Comment (xllx1) [inline src/seedu/addressbook/common/Utils.java:42]: Utils is a good home for the convertToLowerCase method!

File: src/seedu/addressbook/commands/FindCommand.java [hunk 5]
```diff
@@ -1,13 +1,11 @@
 package seedu.addressbook.commands;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```
Comment (ewaldhew) [inline src/seedu/addressbook/commands/FindCommand.java:3]: Collapsing imports is against our coding standard. Make sure you set IntelliJ to not do this: Preferences > Editor > Code Style > Java > Imports tab set Class count to use import with '*'

File: src/seedu/addressbook/common/Utils.java (hunk 3 above)
Comment (ewaldhew) [inline src/seedu/addressbook/common/Utils.java:39]: Good job on pulling this method out and making it more general-purpose, but you missed the header comment :(

File: test/expected.txt [hunk 6]
```diff
@@ -264,37 +271,37 @@
...
-|| 
+||
 || 2 persons listed!
 || ===================================================
 || Enter command: || [Command entered:  clear]
 || Address book has been cleared!
 || ===================================================
 || Enter command: || [Command entered:  list]
-|| 
+||
```
Comment (ewaldhew) [inline test/expected.txt:304]: All these whitespace changes cause the tests to fail and clog up the PR. Do avoid having unrelated changes in a PR, check again before committing is a good practice!

File: src/seedu/addressbook/common/Utils.java (hunk 4 above)
Comment (ewaldhew) [inline src/seedu/addressbook/common/Utils.java:42]: Coding standard requires space around operators, i.e. `for (i = 0; i < 10; i++) {` ← Good `for(i=0;i<10;i++){` ← Bad

**Review by ewaldhew — COMMENTED** [review]
Good job, room for improvement. Close this PR after seeing

**Comment by ewaldhew** [comment]
Please close PRs after reading the review, thanks!
